app/src/main/java/com/keylesspalace/tusky/TabPreferenceActivity.kt

```diff
@@ -261,28 +275,83 @@ class TabPreferenceActivity : BaseActivity(), Injectable, ItemInteractionListene
 
     private fun showSelectListDialog() {
         val adapter = ListSelectionAdapter(this)
+
+        val statusLayout = LinearLayout(this)
+        statusLayout.gravity = Gravity.CENTER
+        val progress = ProgressBar(this)
+        val preferredPadding = getDimensionFromAttribute(this, androidx.appcompat.R.attr.dialogPreferredPadding)
+        progress.setPadding(preferredPadding, 0, preferredPadding, 0)
+        progress.visible(false)
+
+        val noListsText = TextView(this)
+        noListsText.setPadding(preferredPadding, 0, preferredPadding, 0)
+        noListsText.text = getText(R.string.select_list_empty)
+        noListsText.visible(false)
+
+        statusLayout.addView(progress)
+        statusLayout.addView(noListsText)
+
+        val dialogBuilder = AlertDialog.Builder(this)
+            .setTitle(R.string.select_list_title)
+            .setNeutralButton(R.string.select_list_create) { _, _ ->
```

Comment:
This should probably be visible all the time? (currently it is hidden below)

Maybe called "Manage lists"?

Reply:
yes, good idea

Reply:
Done.